src/components/DocExplorer.js

```diff
@@ -221,6 +221,10 @@ class SearchDoc extends React.Component {
       this.props.searchValue !== nextProps.searchValue;
   }
 
+  _isMatch(sourceText, searchValue) {
+    return sourceText.search(new RegExp(searchValue, 'i')) !== -1;
```

Comment:
This will show an uncaught error in console without valid search value I think. Not sure what the best way to handle it would be, but let's at least catch this `invalid regular expression` error.